airflow/utils/context.py

```diff
@@ -322,14 +322,10 @@ class Context(MutableMapping[str, Any]):
     """
 
     _DEPRECATION_REPLACEMENTS: dict[str, list[str]] = {
```

Comment:
Let’s remove this dict entirely in this PR. Not technically related, but it’s better than to potentially forget to do it.

Reply:
removed it